src/polyfills/array-includes.js

```diff
@@ -24,9 +24,11 @@
  */
 function includes(value, opt_fromIndex) {
   const fromIndex = opt_fromIndex || 0;
+  // eslint-disable-next-line local/no-invalid-this
   const len = this.length;
   let i = fromIndex >= 0 ? fromIndex : Math.max(len + fromIndex, 0);
   for (; i < len; i++) {
+    // eslint-disable-next-line local/no-invalid-this
```

Comment:
Question: Given the failure case is broken binary, should this exclusion require a REVIEW or OK flag too?

Might not be necessary, but just wanted to ask for everyone's clarification.